runtime/shared_common/shrinit.h

```diff
@@ -199,6 +199,7 @@ typedef struct J9SharedClassesOptions {
 #define OPTION_TEST_DOUBLE_PAGESIZE "testDoublePageSize"
 #define OPTION_TEST_HALF_PAGESIZE "testHalfPageSize"
 #define OPTION_EXTRA_STARTUPHINTS_EQUALS "extraStartupHints="
+#define OPTION_SHARE_LAMBDAFORM "shareLambdaForm"
```

Comment:
You can put a comment here or in the commit message that this option is an internal option for devs/testing.

Reply:
Done in force push.